permissions/change/grantPermission.ps1

```diff
@@ -144,11 +142,10 @@ function Get-HelloIdTopdeskTemplateById {
     if ([string]::IsNullOrEmpty($entitlementSet.$type)) {
         $message = "Action '$type' for entitlement '$($pRef.id)' is not configured."
         $outputContext.AuditLogs.Add([PSCustomObject]@{
-                Action  = "GrantPermission"
                 Message = $message
                 IsError = $false
             })
-        Throw "Action is not configured"
+        return
```

Comment:
This was a throw before, should this not stay an error? I can't judge this to my lack of knowledge of TOPdesk on this part.